src/containerapp/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.3.0
```

Comment:
Bumped to 0.3.0 since the old containerapp private extension whl files we have been giving customers in the past went up to 0.2.4. We want to avoid confusion by bumping this